agent/api.go

```diff
@@ -42,8 +42,8 @@ func (a *API) SetAuthorID(authorID string) {
 	a.agentAPI.SetCustomHeader("X-Author-Id", authorID)
 }
 
-// ListChats returns chat summaries list.
-func (a *API) ListChats(filters *chatsFilters, sortOrder, pageID string, limit uint) (summary []ChatSummary, found uint, previousPage, nextPage string, err error) {
+// It returns chat info for the chats a Customer participated in.
```

Comment:
```suggestion
// ListChats returns chat info for the chats an Agent has access to.
```
It seems that Customer API comment entered the Agent API code
https://github.com/livechat/lc-sdk-go/blob/a744dd2d9eedcd147a0b0e03128b71034c5ab362/customer/api.go#L135-L136